src/js/disability-benefits/routes.jsx

```diff
@@ -19,6 +21,14 @@ const routes = [
       key="/your-claims/compensation-claims/"
       path="/your-claims/compensation-claims/"/>,
   <Route
+      component={DocumentRequest}
```

Comment:
These two routes probably make more sense underneath the ClaimPage route, since they'll be associated with a claim.